lab_3_generate_by_ngrams/main.py

```diff
@@ -41,6 +43,22 @@ def _tokenize(self, text: str) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(text, str) or not text or not any(char.isalpha() for char in text):
+            return None
+        self.text = text
+        splited_text = text.split()
+        final_text = []
+        for word in splited_text:
+            clear_word = "".join(char.lower() for char in word if char.isalpha())
+            clear_word = list(clear_word)
```

Comment:
join and then list... too much - simplify

Reply:
simplified